Makefile

```diff
@@ -2,37 +2,46 @@ prefix ?= /usr/local
 bindir = $(prefix)/bin
 libdir = $(prefix)/lib
 
-builddir = $(shell ./Scripts/builddir.sh "Release")
+repodir = $(shell pwd)
+builddir = $(repodir)/.build
+flags=-Xlinker -weak-l_InternalSwiftSyntaxParser -Xswiftc -suppress-warnings
 
-build:
-	xcodebuild -scheme muter -configuration Debug > /dev/null 2>&1
+build: 
+	swift build -c debug $(flags)
 
-build-release:
-	xcodebuild -scheme muter -configuration Release > /dev/null 2>&1
+build-release: 
+	swift build -c release --product muter --disable-sandbox $(flags)
 
-release:
-	./Scripts/shipIt.sh $(VERSION)
+project:
+	swift package generate-xcodeproj
+
+release: 
+	./Scripts/shipIt.sh $(version)
 
 install: build-release
-	install -d "$(bindir)"
-	install "$(builddir)/muter" "$(bindir)"
+	install -d "$(bindir)" "$(libdir)"
+	install "$(builddir)/release/muter" "$(bindir)"
 
 uninstall:
-	rm -f "$(bindir)/muter"
+	rm -rf "$(bindir)/muter"
+
+clean:
+	rm -rf .build
 
-run: build-release
-	$(builddir)/muter
+run: build
+	$(builddir)/debug/muter
 
-test: build
-	xcodebuild -scheme muter -only-testing:muterCoreTests test
+test: 
+	swift package generate-xcodeproj
+	@./Scripts/test_only.sh "muterTests"
```

Comment:
What does the `@` sign do? Is that a `make` thing?

Reply:
`@` just dont `echo` the command to `sdtout`